main.py

```diff
@@ -1,12 +1,305 @@
+
+    customer_id = input("Which customer ID would you like to select? ")
+    if customer_id.isnumeric():
+        customer_id = int(customer_id)
+        customer_list.selected_customer = customer_list.get_customer(id=customer_id)
+    else:
+        print("Please enter valid customer ID: ")
+
+    if customer_list.selected_customer:
+        print_squiggle()
+        print("Selected customer: ", customer_list.selected_customer)
+
+    return customer_list.selected_customer["id"]
+
+
+def select_customer(customer_list):
+    print("Here are all of the customers: ")
+    all_customers = customer_list.list_customers()
+
+    for customer in all_customers:
+        print(f"Id: {customer['id']}, name: {customer['name']}")
+
+    select_by = input("What would you like to select by? Enter either a name or an id: ")
+    if select_by == "name":
+        name = input("Which customer name would you like to select? ")
+        customer_list.get_customer(name=name)
+    elif select_by == "id":
+        customer_id = input("Which customer ID would you like to select? ")
+        if customer_id.isnumeric():
+            customer_id = int(customer_id)
+            customer_list.selected_customer = customer_list.get_customer(id=customer_id)
+    else:
+        print("Please enter valid customer ID: ")
+
+    if customer_list.selected_customer:
+        print_squiggle()
+        print("Selected customer: ", customer_list.selected_customer)
+
+    
+
+def delete_customer(customer_list):
+    customer_id = input("Please input the customer ID you would like to delete: ")
+    if customer_id.isnumeric():
+        customer_list.delete_customer(int(customer_id))
+        print_squiggle()
+        print("The customer has successfully been deleted. ")
+        print_squiggle()
+
+def update_customer(customer_list):
+    print(f"Great! Let's update the customer: {customer_list.selected_customer}")
+    name=input("What is the new name of the customer? ")
+    phone=input("What is their new phone number? (xxx-xxx-xxxx) ")
+    postal_code=input("What is their new postal code? ")
+    response = customer_list.update_customer(name=name, phone=phone, postal_code=postal_code)
+    print_squiggle()
+    print("Updated customer:", response["name"])
+
+
+def main(play=True):
+
+    video_list = VideoList(URL)
+    customer_list = CustomerList(URL)
+    rental_list = RentalList(URL)
+    options = list_options()
+
+    while play==True:
+        choice = make_choice(options, video_list, customer_list)
+
+        video_list.print_selected()
+        customer_list.print_selected()
+
+        if choice=='1':
+            create_video(video_list)
+            
+        elif choice=='2':
+            update_video(video_list)
+
+        elif choice=='3':
+            delete_video(video_list)
+        
+        elif choice=='4':
+            print_squiggle()
+            for video in video_list.list_videos():
+                print(video)
```

Comment:
Additionally, you can check if there are videos that exist in the project and if not, direct the user to add videos. This same logic can be used for customers and rentals too.